vegafusion-python/vegafusion/plan_resolver.py

```
@@ -0,0 +1,319 @@
+                return
+
+        for child in _get_child_nodes(variant, inner):
+            self._resolve_external_tables(child, datasets, sidecar)
+
+
+def _extract_table_name(table_ref: Any) -> str:  # noqa: ANN401
+    """Extract table name string from an OwnedTableReference."""
+    which = table_ref.WhichOneof("table_reference_enum")
+    if which == "bare":
+        return str(table_ref.bare.table)
+    elif which == "partial":
```

Ruff (0.15.6)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)